privatek8s.tf

```
@@ -112,7 +112,7 @@ resource "azurerm_kubernetes_cluster_node_pool" "infracipool" {
   orchestrator_version  = local.kubernetes_versions["privatek8s"]
   kubernetes_cluster_id = azurerm_kubernetes_cluster.privatek8s.id
   enable_auto_scaling   = true
-  min_count             = 0
+  min_count             = 1
```

I would stay at 0 for this nodepool for now as we never encounter the autoscaler problem and would save 33$/month for now
```suggestion
  min_count             = 0
```

---

this is the case we hit it on, so you may hit it

---

we plan to migrate all the agent workload to arm64 so we plan to let the 0 minimum for the non arm64 nodes